lib/core/presentation/routes.dart

```diff
@@ -332,6 +333,7 @@ final List<dynamic> pages = [
   MenuItem(
       title: "UI Kits (Clones)",
       items: [
+        SubMenuItem('Youtube Clone App', HomePage(), path: HomePage.path),
```

Comment:
Name should be something like YoutubeClonePage() instead of HomePage(), its confusing.

Reply:
Keep it under misc section. UI kits clones are the sections for such UIs that have implemented multiple pages of an existing app or UI kit